dummy_search/setup.py

```diff
@@ -0,0 +1,32 @@
+# This file is part of the CERN Indico plugins.
+# Copyright (C) 2014 - 2021 CERN
+#
+# The CERN Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License; see
+# the LICENSE file for more details.
+
+from setuptools import find_packages, setup
+
+
+setup(
```

Comment:
Let's also use the new structure - see `vc_zoom/setup.{py,cfg}` for an example. Also, version should be 3.0-dev.

Reply:
Good idea, but shouldn't this be based on 2.3 instead, as the search interface?

Reply:
That's what I was saying during our meeting - with 3.0 progressing well and deployment being no longer an issue (I originally thought it might require much deeper infrastructure changes, like moving to CentOS8), we can ditch the plan of keeping the new search compatible with 2.3 and just rebase it on top of master.

And for the core all that needs to be done to use the new search logic on Python 3 is removing the now obsolete `__future__` imports from the files in ` indico/modules/search`

Reply:
My bad, I recall that - completely forgot this PR meant to include that too.